backend/app.js

```diff
@@ -16,6 +17,7 @@ app.get('/', (req, res) => {
 });
 
 app.use('/api/v1/auth', userRoutes);
+app.use('/api/v1/products', checkAuth, voteRoutes);
```

Comment:
the feature looks good to me though was wondering if we could join these routes

Reply:
this means deleting the voteRoutes file and transferring the route to productsRoute File